docs/source/tutorial-model-registry.rst

```diff
@@ -0,0 +1,110 @@
+
+.. code-block:: python
+
+    # ... `run_id` and `model_uri` as above
+    model_uri = f"runs:/{run_id}/model"
+    name = "Wine Quality"
+    tags = {"run_id": run_id, "sem_ver": "0.1.0"}
+    registered_model = mlflow.register_model(model_uri=model_uri, name=name, tags=tags)
+    print(f"Name: {registered_model.name}, Version: {registered_model.version}")
+
+Model lifecycle
+----------------------------------
+
+The MLflow Registry has a simple model lifecycle. A particular version of the model may be in one of the following states:
+
+- ``None``: This version of the model has been registered with the Model Registry but hasn't been staged for deployment. If broad testing fails at this stage, the model should be archived.
+- ``Staging``: The model version has been staged for deployment. This is typically a short-lived stage, as the version is quickly deployed or rejected.
+- ``Production`` : The version deployed for use. 
+- ``Archived``: This version of the model is no longer deployed. It is available for record-keeping, auditing, post-mortem analysis, and so forth.
+
+.. image :: _static/images/tutorial-model-registry/registry-lifecycle.png
+   :alt: Model lifecycle
+
+It is possible to transition from any state to any other state (e.g., ``None`` directly to ``Production``) but the above diagram shows the intended flow: models are registered and then staged and then deployed. If a model is not moving towards deployment, it should be archived.
+
+Model versions
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you register a model with a given ``name``, the registry assigns a new version number to that model. This is an integer that increments by one when a model is registered. Just as with the lifecycle model, this is intentionally simple. Your organizational MLOps lifecycle, versioning scheme, approval process, and so forth will almost certainly be more complex.
+
+**Tags** are a dictionary of key-value pairs that you can associate with a model version. Among other uses, you can use these to associate a specific run or model with aspects of your more complex lifecycle. In the sample above, for instance, the ``sem_ver`` tag is used to associate a semantic version with the model version. Determining the proper value for such a tag is up to you and your organization.
+
+Transitioning between lifecycle states
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can transition a model version from one state to another using the MLflow UI or the MLflow API. In the UI, click a model version to open the details page associated with the version. Select the **Stage** dropdown and choose the to-be-entered state. 
+
+In the confirmation dialog, you'll see a checkbox that is selected. You generally only want to have one version at a time in the ``Staging`` or ``Production`` states. If you check this box, the registry will automatically transition any other versions in the to-be-entered state to the ``Archived`` state. This is the recommended behavior.
```

Comment:
(with the exception of certain advanced deployment considerations: canary, multi-armed bandit, shadow deployment, etc..) where multiple "production" states can be active at a time.